monitor/indexingStatus.js

```diff
@@ -100,268 +100,167 @@ export async function generateEndpoints(data) {
   }
 }
 
-export async function queryDecentralizedIndex(hostedEndpointToDecenNetwork) {
+export async function queryDecentralizedIndex(data) {
   if (!process.env.GRAPH_API_KEY) {
```

Comment:
I don't think we need the graph api key anymore, can be removed from the env vars too

Reply:
Okay I see it is used elsewhere, nvm